lessons/Early-Math/absolute-value-and-opposites/langs/en-us/index.adoc

```diff
@@ -75,15 +79,23 @@ The function used to represent "the opposite" is called `negate`. It can be used
 [.image]
 @show{(coe '(negate 2))}
 
+@slidebreak
+
 @QandA{
 @Q{What happens if we nest a negating Circle of Evaluation _inside of_ another negating Circle?}
 @A{The opposite of the opposite of a number is the number itself.}
 }
+
 [.image]
 @show{(coe '(negate (negate 2)))}
 
-- Try negating a few more times, to make sure you've got the hang of it. What do each of these Circles evaluate to?
+@slidebreak
+
+@QandA{
+@Q{Try negating a few more times, to make sure you've got the hang of it. What do each of these Circles evaluate to?}
+}
 
+@ifnotslide{
```

Comment:
Last time I tried to put it on a slide, I got a whole bunch of COEs stacked on top of one another that I had to reposition and resize. I'm assuming that is no longer the case? What am I supposed to put here so that it builds appropriately in slides?